src/bcd.c

```diff
@@ -862,11 +862,11 @@ bcd_backtrace_thread(struct bcd_session *session)
 	char *tp = NULL;;
 	pid_t tid = session->tid;
 	ssize_t r;
-	size_t delta = 2;
+	size_t delta = 0;
 
-	u.cargv[0] = bcd_config.invoke.path;
+	u.cargv[delta++] = bcd_config.invoke.path;
 
-	r = bcd_arg_get(&(u.argv[delta]), sizeof(u.argv) / sizeof(*u.argv) - delta,
+	r = bcd_arg_get(&(u.argv[delta]), sizeof(u.argv) / sizeof(*u.argv) - 2,
```

Comment:
This is the same functionally, and I agree the code deserves some clean up. IMO, we should fix the immediate  issue and then we can follow up with an audit after discussion.